frontend/integration-tests/tests/dashboards/project-dashboard.scenario.ts

```diff
@@ -180,6 +180,7 @@ describe('Project Dashboard', () => {
       // refresh browser to reload launcher links
       await browser.refresh();
       await dashboardView.isLoaded();
+      await browser.wait(until.visibilityOf(projectDashboardView.launcherCard));
       expect(projectDashboardView.launcherCard.isDisplayed()).toBe(true);
```

Comment:
This line can be removed since the line above will fail if the card isn't displayed.

Reply:
Removed.